cli/src/cli/actions/auditRulesAction.ts

```
@@ -0,0 +1,105 @@
+      const confirmVulnerableFiles = await confirm({
+        message: `\n Do you want to clean these files? (will remove all non-ASCII characters)`,
+      });
+
+      if (isCancel(confirmVulnerableFiles)) {
+        process.exit(0);
+      }
+
+      if (confirmVulnerableFiles) {
+        for (const file of vulnerableFiles) {
+          let text = readFileSync(file).toString();
+          writeFileSync(file, text.replace(/[^\x00-\x7F]/g, ''));
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)